config/common.mk

```diff
@@ -87,6 +87,9 @@ export VOICE_DETECTOR_EN ?= 0
 # 1 to use 8K sample rate for VAD, 0 to use 16K sample rate for VAD
 VAD_USE_8K_SAMPLE_RATE ?= 0
 
+# 1 to enable the blue flashing light when connected, 0 to disable
+CONNECTED_BLUE_LIGHT ?= 0
```

Comment:
Since this line disables the flashing by default, I think the README should be updated to reflect the option. Maybe just add a note in there that says something like:

> Firmware can be compiled to either blink blue LEDs to indicate connected buds or keep them off when connected. Build with 
> `CONNECTED_BLUE_LIGHT=1 ./build.sh` to enable blinking when connected, or
> `CONNECTED_BLUE_LIGHT=0 ./build.sh` (or just the default `./build.sh`) to keep the LEDs off when connected